envoy/admin/v2/admin.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package envoy.admin;
+
+import "google/protobuf/any.proto";
+import "envoy/api/v2/rds.proto";
+
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: ConfigDump]
```

Comment:
I wouldn't worry about RST docs right now for this. There is a generally open item for how to document APIs in RST. Right now we don't really do it. We just document configuration. (In any case I don't think it's critical that these protos have HTML docs right now). We can figure it out in the context of https://github.com/envoyproxy/data-plane-api/issues/351.

Reply:
Sounds good. I didn't realize we weren't generating docs for the APIs, good that's it's on the list.